src/vmm/src/builder.rs

```diff
@@ -864,7 +875,13 @@ pub fn configure_system_for_boot(
     {
         for vcpu in vcpus.iter_mut() {
             vcpu.kvm_vcpu
-                .configure(vmm.guest_memory(), entry_addr)
+                .configure(
+                    vmm.guest_memory(),
+                    entry_addr,
+                    Aarch64CpuConfiguration {
+                        regs: HashMap::new(),
+                    },
+                )
```

Comment:
You pass the CpuConfiguration, but where will the CpuTemplate come from?
Also this "query registers first, apply template later" approach really does not for arm, because you don't know what registers to query in the first place.

Reply:
The registers to query will be based on the addresses in the template. The CPU template will come from the /cpu-config API call which is a different call path. This is a place holder for now to demonstrate where the CPU configuration will be passed into.